openwisp_controller/pki/apps.py

```diff
@@ -11,3 +13,18 @@
 class PkiConfig(DjangoX509Config):
     name = 'openwisp_controller.pki'
     verbose_name = _('Public Key Infrastructure')
+
+    def __setmodels__(self):
+        self.cert_model = load_model('django_x509', 'cert')
+
+    def ready(self):
+        super().ready()
+        self.__setmodels__()
+        self.connect_signals()
+
+    def connect_signals(self):
+        post_save.connect(
+            self.cert_model.post_save,
+            sender=self.cert_model,
+            dispatch_uid='cert.post_save',
+        )
```

Comment:
same here, this code belong to `config`, it's a problem with the config checksum, not with pki